### PR TITLE
Added SameWave to user data.

### PR DESCRIPTION
**Pull request by MondeSameWave**

### CODE REVIEW

**Comment by wifelette** [comment]
Looks good! Can you remove the second sentence from your description? Should be good to merge from there!

**Comment by wifelette** [comment]
Sorry, spoke too soon :p The logo isn't part of this PR. Gotta add that too.

File: data/users.yml [hunk 1]
```diff
@@ -815,6 +815,11 @@
   image: airhelp.png
   alt: "AirHelp"
   use: "AirHelp web app is built with Ember"
+- url: http://www.samewave.com/
+  image: samewave.png
+  alt: "SameWave"
+  use: "We use ember together with cordova to power our v2 hybrid ios and android app. Our app allows users to easily track and talk about the important stuff with a scoreboard for the whole team."
```
Comment (locks) [inline data/users.yml]: can you capitalize Ember, Cordova, iOS and Android?

**Comment by MondeSameWave** [comment]
Was informed by our design department that our logo has changed. Will wait for assets to be updated then resume PR. @wifelette Thanks for your assistance.

**Pull request closed without merging**